app/scripts/modules/core/src/managed/ArtifactRow.module.css

```diff
@@ -1,22 +1,27 @@
 .ArtifactRow {
-  height: 48px;
+  height: 56px;
```

Comment:
@gcomstock fyi, I cranked up the height on these rows because with the extra noise and weight of the environment bars it was looking _very_ tight. LMK if you would like to tweak it further or go back to a tighter list.

Reply:
Sounds good! Thanks for keeping the measurements divisible by 8